aea/helpers/dependency_tree.py

```diff
@@ -160,7 +152,6 @@ def _resolve_tree_aux(
             if root_package in visited:
                 # cycle found; raise error
                 cls._raise_circular_dependency_error(root_package, stack)
-                return
```

Comment:
impossible to cover this line without needless gymnastics since there is a raise condition